widget/titter.jsx

```diff
@@ -1,12 +1,5 @@
 var React = require('react');
 
-    domain = '';
-    if (process.env.DOMAIN == undefined) {
-        domain = 'wrioos.com';
-    } else {
-        domain = process.env.DOMAIN;
-    }
-
```

Comment:
Please leave all cross-domain code intact, wrio applications should work on any configured domain.